File: frb_example/pure_dart/lib/src/rust/api/primitive_list_sync_misc.dart
// ignore_for_file: invalid_use_of_internal_member, unused_import

import '../frb_generated.dart';
import 'package:flutter_rust_bridge/flutter_rust_bridge_for_generated.dart';
import 'primitive_list_misc.dart';

ZeroCopyVecOfPrimitivePackTwinNormal handleZeroCopyVecOfPrimitiveSyncTwinNormal(
        {required int n, dynamic hint}) =>
    RustLib.instance.api
        .handleZeroCopyVecOfPrimitiveSyncTwinNormal(n: n, hint: hint);

File: frb_example/pure_dart/rust/src/api/primitive_list_sync_misc.rs
use super::primitive_list_misc::*;
use flutter_rust_bridge::*;

#[frb(sync)]
pub fn handle_zero_copy_vec_of_primitive_sync_twin_normal(
    n: i32,
) -> ZeroCopyVecOfPrimitivePackTwinNormal {
    handle_zero_copy_vec_of_primitive_twin_normal(n)
}
